luwak/src/main/java/org/apache/lucene/search/spans/SpanRewriter.java

```diff
@@ -94,14 +94,14 @@ protected Query rewriteTermsQuery(TermsQuery query) {
             PrefixCodedTerms.TermIterator it = terms.iterator();
             for (int i = 0; i < terms.size(); i++) {
                 BytesRef term = BytesRef.deepCopyOf(it.next());
-                if (spanQueries.containsKey(it.field()) == false) {
+                if (!spanQueries.containsKey(it.field())) {
```

Comment:
Hm, this one I actually disagree with - I think the ```!``` is easy to miss squashed up against the opening parenthesis, whereas the explicit ```== false``` is obvious.